### PR TITLE
Update Essentials Accelerometer API Docs

### PR DESCRIPTION
**Pull request by jfversluis**
### Description of Change

Moves the external comments to inline and minor improvements.

### Issues Fixed

Related to #3960

### CODE REVIEW

**Review by jsuarezruiz — APPROVED** [review]

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 1]
```diff
@@ -5,102 +5,187 @@
 
 namespace Microsoft.Maui.Devices.Sensors
 {
+	/// <summary>
+	/// Accelerometer data of the acceleration of the device in three dimensional space.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:9]:
```suggestion
	/// Accelerometer data of the acceleration of the device in three-dimensional space.
```

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 2]
```diff
@@ -5,102 +5,187 @@
...
+	/// </summary>
 	public interface IAccelerometer
 	{
+		/// <summary>
+		/// Event triggered when reading of sensor changes.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:14]:
```suggestion
		/// Occurs when the sensor reading changes.
```

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 3]
```diff
@@ -5,102 +5,187 @@
...
+		/// </summary>
 		event EventHandler<AccelerometerChangedEventArgs>? ReadingChanged;
 
+		/// <summary>
+		/// Event triggered when a shake has been detected on the device.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:19]:
```suggestion
		/// Occurs when the accelerometer detects that the device has been shaken.
```

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 4]
```diff
@@ -5,102 +5,187 @@
...
+		/// </summary>
 		event EventHandler? ShakeDetected;
 
+		/// <summary>
+		/// Gets if reading the accelerometer is supported on this device.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:24]:
```suggestion
		/// Gets a value indicating whether reading the accelerometer is supported on this device.
```

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 5]
```diff
@@ -5,102 +5,187 @@
...
+		/// </summary>
 		bool IsSupported { get; }
 
+		/// <summary>
+		/// Gets if accelerometer is being monitored.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:29]:
```suggestion
		/// Gets a value indicating whether the accelerometer is being monitored.
```

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 6]
```diff
@@ -5,102 +5,187 @@
...
 	}
 
-	/// <include file="../../docs/Microsoft.Maui.Essentials/Accelerometer.xml" path="Type[@FullName='Microsoft.Maui.Essentials.Accelerometer']/Docs/*" />
+	/// <summary>
+	/// Accelerometer data of the acceleration of the device in three dimensional space.
+	/// </summary>
 	public static class Accelerometer
 	{
+		/// <summary>
+		/// Event triggered when reading of sensor changes.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:54]:
```suggestion
		/// Occurs when the accelerometer reading changes.
```

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 7]
```diff
@@ -5,102 +5,187 @@
...
+		/// </summary>
 		public static event EventHandler<AccelerometerChangedEventArgs> ReadingChanged
 		{
 			add => Default.ReadingChanged += value;
 			remove => Default.ReadingChanged -= value;
 		}
 
+		/// <summary>
+		/// Event triggered when a shake has been detected on the device.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:63]:
```suggestion
		/// Occurs when the accelerometer detects that the device has been shaken.
```

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 8]
```diff
@@ -5,102 +5,187 @@
...
 
+		/// <summary>
+		/// Gets if reading the accelerometer is supported on this device.
+		/// </summary>
 		public static bool IsSupported
 			=> Default.IsSupported;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/Accelerometer.xml" path="//Member[@MemberName='IsMonitoring']/Docs/*" />
+		/// <summary>
+		/// Gets if accelerometer is being monitored.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:78]:
```suggestion
		/// Gets a value indicating whether the accelerometer is being monitored.
```

File: src/Essentials/src/Accelerometer/Accelerometer.shared.cs [hunk 9]
```diff
@@ -5,102 +5,187 @@
...
+		/// Event triggered when a shake has been detected on the device.
+		/// </summary>
 		public static event EventHandler ShakeDetected
 		{
 			add => Default.ShakeDetected += value;
 			remove => Default.ShakeDetected -= value;
 		}
 
+		/// <summary>
+		/// Gets if reading the accelerometer is supported on this device.
```
Comment (hartez) [inline src/Essentials/src/Accelerometer/Accelerometer.shared.cs:72]:
```suggestion
		/// Gets a value indicating whether reading the accelerometer is supported on this device.
```

**Review by hartez — CHANGES_REQUESTED** [review]
Looks good, some suggestions.

**Review by hartez — APPROVED** [review]

**Pull request merged**